src/Aspire.Dashboard/Components/ResourcesGridColumns/EndpointsColumnDisplay.razor

```diff
@@ -72,11 +72,11 @@ else
                     Endpoints
                 </Header>
                 <Body>
-                    <div style="max-height:400px; overflow-x:hidden; overflow-y: auto; padding:5px 10px;">
+                    <div class="endpoint-popup">
                         @foreach (var item in items)
                         {
                             var d = (DisplayedEndpoint)item.Data!;
-                            <div style="margin-bottom: 4px;">
+                            <div class="endpoint-link">
                                 @if (d.Url != null)
                                 {
                                     <a href="@d.Url" target="_blank">@d.Text</a>
```

Comment:
If it can be truncated now, should we have a tooltip with the full value of `d.Text`?